README.md

```diff
@@ -331,7 +331,8 @@ npm run build
 For an easier and faster setup of your local environment, run:
 
 ```bash
-node dist/cli.js setup <binaries>
+❯ cd <zombinet project dir>/javascript/packages
```

Comment:
I have the following directory tree structure
```
> javascript % tree -L 3 -I node_modules -d

.
└── packages
   ├── cli
   │   ├── dist
   │   └── src
   ├── orchestrator
   │   ├── dist
   │   ├── src
   │   └── static-configs
   └── utils
        ├── dist
        └── src
```
where `cli.js` is in `cli/dist/cli.js`.

When i am trying to execute `node dist/cli.js version` from `javascript` or `javascript/packages` i've got error message `Error: Cannot find module '/zombienet/javascript/dist/cli.js`. If i try to call `node cli/dist/cli.js version` from `javasript` dir i get the same error. However if to call `node cli/dist/cli.js version` from `javascript/packages` i get the current cli version.

Reply:
as far as i understood `cli.ts` was [moved](https://github.com/paritytech/zombienet/blob/3a9a10610592bcf380a4007a7d4517b3e33769bc/javascript/packages/cli/src/cli.ts) from src/cli.ts to packages/cli/cli.ts with this [PR](https://github.com/paritytech/zombienet/pull/475/files). It seems for me that we need to update the previosly working call node `dist/cli.js` with `node cli/dist/cli.js`. And we have to do this from `packages` directory as currently `cli/dist/cli.js` is located in `packages`

Reply:
this comment relates to all similar changes around the project